test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -4010,15 +4010,17 @@ BOOST_AUTO_TEST_CASE(call_forward_bytes_length)
 	compileAndRun(sourceCode, 0, "sender");
 
 	// No additional data, just function selector
-	ABI_CHECK(callContractFunction("viaCalldata()"), encodeArgs(4));
+	ABI_CHECK(callContractFunction("viaCalldata()"), encodeArgs(0x20));
+	// Should be this with 0.5.0: encodeArgs(4));
 	ABI_CHECK(callContractFunction("viaMemory()"), encodeArgs(0x20));
 	// Should be this with 0.5.0: encodeArgs(4));
 	ABI_CHECK(callContractFunction("viaStorage()"), encodeArgs(0x20));
```

Comment:
Aren't these need to return 4?

Reply:
No, this will be done in the other pull request. This is the pull request that adds padding for encoding from calldata to be identical to the encoding from all other types (this applies to `call` and to `abi.encodePacked`). There is another pull request that changes the behaviour of `call` to not use padding for anything.